blazor/pdfviewer-2/server-deployment/deploy-maui-windows.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Deploy SfPdfViewer in Blazor MAUI in windows | Syncfusion
+description: Checkout and learn here all about how to deploy SfPdfViewer in Blazor MAUI in windows in Syncfusion Blazor SfPdfViewer component and much more details.
+platform: Blazor
+control: SfPdfViewer
+documentation: ug
+---
+
+# How To Deploy SfPdfViewer in Blazor MAUI in windows
+
+This section briefly explains about how to deploy [Blazor SfPdfViewer](https://www.syncfusion.com/blazor-components/blazor-pdf-viewer) component in your Blazor MAUI in windows App using Visual Studio.
+
+## Prerequisites
+
+To use the MAUI project templates, install the Mobile development with .NET extension for Visual Studio. For more details, refer [here](https://docs.microsoft.com/en-us/dotnet/MAUI/get-started/installation).
+
+## Deploy SfPdfViewer into Blazor MAUI in windows application
+
+1. Start Visual Studio and select **Create a new project**.
+
+2. For a Blazor MAUI application experience, choose the **.NET MAUI Blazor App** template. Select **Next**. 
+![Create-new-blazor-server-app](images/start-window-create-new-project.png)
+
+3. Provide a **Project Name** and confirm that the *Location* is correct. Select Next. 
+![Set-project-name](images/Set-project-name-window.png)
+
+4. In the **Additional information** dialog, set the target framework and create the project.
+![Set-target-framework](images/Additional_information.png)
+
+## Install Blazor SfPdfViewer NuGet package in Blazor MAUI in windows application
+
+Syncfusion Blazor components are available in [nuget.org](https://www.nuget.org/packages?q=syncfusion.blazor). To use Syncfusion Blazor components in the application, add reference to the corresponding NuGet. Refer to [NuGet packages topic](https://blazor.syncfusion.com/documentation/nuget-packages) for available NuGet packages list with component details and [Benefits of using individual NuGet packages](https://blazor.syncfusion.com/documentation/nuget-packages#benefits-of-using-individual-nuget-packages).
+
+Install the [Syncfusion.Blazor.SfPdfViewer](https://www.nuget.org/packages/Syncfusion.Blazor.SfPdfViewer) NuGet package as a reference to your project from the [NuGet.org](https://www.nuget.org/packages?q=syncfusion.blazor).
+
+![Install the PDF Syncfusion.Blazor.SfPdfViewer package](images/nuget-package-windows.png)
+
+## Register Syncfusion Blazor Service
+
+Open **~/_Imports.razor** file and import the **Syncfusion.Blazor.SfPdfViewer** namespaces.

```

Comment:
Open the **~/_Imports.razor** file and import the **Syncfusion.Blazor.SfPdfViewer** namespaces.